File: backend_samples/library/java/AzureAIVisionFaceDeviceAttestation/src/main/java/com/azure/ai/vision/face/deviceattestation/IsoTime.java
package com.azure.ai.vision.face.deviceattestation;

import java.time.Instant;
import java.time.format.DateTimeFormatter;

/** ISO-8601 UTC timestamp helpers (always ends in "Z"). */
public final class IsoTime {

    public static String now() {
        return DateTimeFormatter.ISO_INSTANT.format(Instant.now());
    }

    public static String from(Instant value) {
        return DateTimeFormatter.ISO_INSTANT.format(value);
    }

    private IsoTime() {
    }
}

File: backend_samples/library/dotnet/AzureAIVisionFaceDeviceAttestation/src/IsoTime.cs
namespace Azure.AI.Vision.Face.DeviceAttestation;

/// <summary>ISO-8601 UTC timestamp helpers (round-trip "o" format, always "Z").</summary>
internal static class IsoTime
{
    public static string Now() => DateTime.UtcNow.ToString("o");

    public static string From(DateTimeOffset value) => value.UtcDateTime.ToString("o");
}
